### ISSUE TITLE
SCSSLint::Location error linting Compass file

### ISSUE THREAD

**Issue by moubry** [causes/scss-lint#336]
When I run `scss-lint` out of the box like this, it works great:

```
$ scss-lint public/assets/web2/scss
```

When I add this configuration file, adding the path to our SCSS files:

```
scss_files: 'public/assets/web2/scss/**/*.scss'
```

I get the following error:

```
$ scss-lint
SCSSLint::Linter::SpaceAfterComma raised unexpected error linting file vendor/bundle/ruby/2.0.0/gems/compass-0.12.2/frameworks/compass/stylesheets/compass/css3/_text-shadow.scss: 'Length must be more than 0, passed 0'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/location.rb:14:in `initialize'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:57:in `new'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:57:in `location_from_range'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:149:in `extract_location'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:40:in `add_lint'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter/space_after_comma.rb:81:in `block in check_commas_after_args'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter/space_after_comma.rb:68:in `each'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter/space_after_comma.rb:68:in `check_commas_after_args'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter/space_after_comma.rb:50:in `check_invocation'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter/space_after_comma.rb:22:in `visit_script_funcall'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:127:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:138:in `block in visit_children'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:136:in `each'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:136:in `visit_children'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:127:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:138:in `block in visit_children'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:136:in `each'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:136:in `visit_children'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:36:in `block in visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter/space_after_comma.rb:8:in `visit_mixindef'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:36:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:127:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:138:in `block in visit_children'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:136:in `each'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:136:in `visit_children'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/sass-3.4.9/lib/sass/tree/visitors/base.rb:38:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:127:in `visit'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/linter.rb:21:in `run'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/runner.rb:58:in `run_linter'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/runner.rb:39:in `block in find_lints'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/runner.rb:37:in `each'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/runner.rb:37:in `find_lints'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/runner.rb:21:in `block in run'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/runner.rb:20:in `each'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/runner.rb:20:in `run'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/cli.rb:52:in `scan_for_lints'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/cli.rb:46:in `act_on_options'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/lib/scss_lint/cli.rb:25:in `run'
/usr/local/var/rbenv/versions/2.1.5/lib/ruby/gems/2.1.0/gems/scss-lint-0.32.0/bin/scss-lint:6:in `<top (required)>'
/usr/local/var/rbenv/versions/2.1.5/bin/scss-lint:23:in `load'
/usr/local/var/rbenv/versions/2.1.5/bin/scss-lint:23:in `<main>'
Report this bug at https://github.com/causes/scss-lint/issues
```

There's a few things going on here, I'm not sure what the issue is.

Thank you for creating `scss-lint` — it's exactly what we needed. So great.

**Comment by moubry** [comment]
Nevermind. It's working. I was in a funky out-of-date Gem situation.

**Comment by mmuneer** [comment]
@moubry do you remember how you fixed this? we just upgraded to rails4 and seeing this again.
#yourFormarbuddy... :)

**Comment by moubry** [comment]
@mmuneer I honestly have no idea. Maybe make sure you're running it in the context of the bundle `bundle exec scss-lint`. If that doesn't work, perhaps uninstalling all gems `gem uninstall --all` and trying again?
